ui/src/link_table_cell_type.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { S } from 'h2o-wave'
+import React from 'react'
+import { XLink } from './link'
+
+export interface LinkTableCellType {
+  /** An identifying name for this component. */
+  name: S
+  /** Where to display the link. Setting this to an empty string or `'_blank'` opens the link in a new tab or window. */
+  target?: S
+}
+
+export const XLinkTableCellType = ({str, target}: { str: S, target?: S}) => {
```

Comment:
Nit: Let's reuse the `LinkTableCellType` interface here instead of a standalone `target` prop.

Reply:
I just pushed, not sure if that's what you meant.

Reply:
Yes, looking good.